cmd/ui/src/views/Explore/EdgeInfo/EdgeObjectInformation.tsx

```diff
@@ -67,7 +67,11 @@ const EdgeObjectInformation: FC<{ selectedEdge: NonNullable<SelectedEdge> }> = (
     } else {
         formattedObjectFields = [
             ...formattedObjectFields,
-            ...formatObjectInfoFields(cypherResponse.edges[0]?.properties || {}),
+            ...formatObjectInfoFields(
+                { ...cypherResponse.edges[0]?.properties, lastseen: selectedEdge.data.lastseen } || {
+                    lastseen: selectedEdge.data.lastseen,
```

Comment:
is the object following the || operator unreachable? it looks like every case sets `formattedObjectFields` to ```formattedObjectFields = [
            ...formattedObjectFields,
            ...formatObjectInfoFields({ lastseen: selectedEdge.data.lastseen }),
        ];
```

if this is accurate, then maybe you could just invert the isError check, and instead check for if !isError, something roughly like:

```
const formattedObjectFields = [
        ...formattedObjectFields,
        ...formatObjectInfoFields({ lastseen: selectedEdge.data.lastseen }),
    ];

    if (!isError) {
        formattedObjectInfoFields.push(...cypherResponse.edges[0].properties)
    } 
```

Reply:
maybe could grab the isSuccess field from useQuery instead of isError, if the logic is updated like this^^

Reply:
This does clean things up a bit so I went ahead and updated the error handling. One caveat is that the lastseen property would be duplicated as originally suggested so I had to delete the property from the cypherResponse if the call went through successfully